metabase/metabase
# revert new tooltips for non-stacked series

[Product doc](https://www.notion.so/metabase/Offer-automated-percentages-on-stacked-charts-a8af42d2b81e458b827b679dac2b61c6)
Epic: https://github.com/metabase/metabase/issues/26891

## Changes

We decided to show the new tooltips only on stacked charts and keep the old tooltip for all other cases. This PR reverts it only for the row chart.

## How to verify

- create stacked row charts GUI question
- ensure it shows the new tooltip on the hover
- convert the GUI question to native, ensure it shows the same new tooltip
- remove stacking and ensure it shows the old tooltip
